engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/UserTaskTransformerTest.java

```diff
@@ -59,8 +59,9 @@ Stream<Arguments> assignees() {
             Arguments.of(null, null),
             Arguments.of("", null),
             Arguments.of(" ", null),
-            Arguments.of("frodo", "frodo"),
-            Arguments.of("=ring.bearer", "ring.bearer"));
+            Arguments.of("frodo", "\"frodo\""),
+            Arguments.of("=ring.bearer", "ring.bearer"),
+            Arguments.of("12345678", "\"12345678\""));
```

Comment:
:thought_balloon: Just adding a clarifying comment: the expected change from `Arguments.of("frodo", "frodo")` to `Arguments.of("frodo", "\"frodo\"")` isn't a breaking one. The parsed expression is an intermediate result that is eventually resolved to the same static expression.

Reply:
Thanks for making this explicit, @koevskinikola 👍